cfme/cloud/keypairs.py

```diff
@@ -53,6 +51,7 @@ class KeyPairEntities(View):
 class KeyPairDetailsEntities(View):
     breadcrumb = BreadCrumb()
     title = Text('//div[@id="main-content"]//h1')
+    table = Table("//div[@id='list_grid']//table")
```

Comment:
It's about time to convert this to use the new Entities conditional view that @izapolsk implemented, which will get you table + tiles + quadicons in a conditional view.

I don't think it needs to be done in the scope of the PR, but @sbulage could you write a card/issue so we don't miss it please?